### DISCUSSION TITLE
“Dual RX Respond” as default choice for “RxMode” + Wideband = not good!

### DISCUSSION THREAD

**Discussion by muppix** [egzumer/uv-k5-firmware-custom#208] [Ideas]
Yesterday I flashed my newly arrived UV-K5 with this excellent custom firmware and was mightily impressed, but soon found myself in a sticky situation as a result of some default settings. I appreciate this is an extreme edge case but thought I’d share anyway. Here’s the two factors which combined to create an undesired situation:

1. The radio was in “Dual RX Respond” mode by default, meaning that a signal on either VFO would ‘capture’ subsequent PTT for a few seconds regardless of which VFO was set as primary. I don’t know if the standard firmware has an equivalent Dual Watch / RxMode setting or not, because I couldn’t wait to try this firmware that everyone’s talking about.
2. I live on a small island and often monitor marine channels (coast guard, harbour operations, etc) on the secondary VFO while using the main VFO to work local repeaters and/or personal nodes and gateways. After flashing release 0.19 I was unexpectedly able to transmit on marine VHF.

You can probably guess what happened here. I was in QSO with several stations via our local repeater on VFO 1 when the harbour master’s transmission briefly broke squelch on VFO 2. I thought I was replying to the local repeater when in fact I was replying to the harbour master on a band I wasn’t licensed to use. Not an ideal situation.

Ultimately the fault was mine because I didn’t read the full wiki and release notes, then again I think that most people would be in a similar situation, and that “Dual RX Respond” combined with the ability to transmit out of band is a bad combination of defaults. Maybe give folks a safety net by only enabling amateur bands and setting “Main TX Dual RX” as the default for RxMode?

**Comment by G0OMK** [comment]
Hi Mark, Yes I think as time goes on we will discover this sort of thing and of course rectify the issues.  This is MENU related and not a firmware issue.  Like anything new in our hands, programming can be confusing as well as a learning curve.  Thanks for the thumbs up, I will bear this in mind, and pass on the info to other users.  73 Dave G0OMK

**Comment by ErikS-web** [comment]
The settings are _personal_ and _customizable_.
But I agree with your observation.

_These are the adjustments I make as standard after flashing a radio..._

31 Mic - microphone sensitivity (+1.1dB .. +15.1dB). Set it to 15.1
32 MicBar - microphone bar that appears while transmitting. Set it to ON
33 ChDisp - channel display style (name + frequency or number). Set it to Name + Freq
34 POnMsg - Power on message. Set it to Message
35 BatTxt - additional battery value on the status bar in % or V(oltage). Set it to Voltage
36 BackLt - backlight duration (OFF/ON/configured time). Set it to 2 minutes
56 AM Fix - activates the autogain AM fix function. Set it to ON
59 RxMode - determine how the upper and lower frequencies are used. Set it to **MAIN TX DUAL RX** - always broadcasts on the primary station (marked ►), listens to both (DW on screen)

and in the hidden menu

67 BatCal - battery calibration, measure the voltage at the back of the radio with a voltmeter and adjust the value in the menu accordingly
68 BatTyp - Battery type, 1600 mAh and 2200 mAh battery have very different discharge curve, this is used to calculate battery level percentage. Set it to 1600, or 2200 (mostly UV5RPlus).
​
p.s. see also Issue #124

**Comment by egzumer** [comment]
You should write to Quansheng then. What this firmware changes with RxMode, is it consolidates 2 options into one (2 options times two settings in each gives you 4 settings in RxMode). Under the hood nothing changes, stock firmware does the same thing. The settings you had were what was there from the factory. Irrelevant what the firmware was, the outcome would be exactly the same.

**Reply by G0OMK** [reply]
Loving the Firmware 0.19 Thank you :)
